plugins/modules/device_credential_workflow_manager.py

```diff
@@ -1961,7 +1966,7 @@ def get_want_assign_credentials(self, AssignCredentials):
         site_id = []
         for site_name in site_name:
             siteId = self.get_site_id(site_name)
```

Comment:
Can we rename from siteId to current_site_id
 current_site_id = self.get_site_id(name)